yt_dlp/extractor/bilibili.py

```diff
@@ -689,10 +683,15 @@ def _real_extract(self, url):
         old_video_id = format_field(aid, None, f'%s_part{part_id or 1}')
         cid = traverse_obj(video_data, ('pages', part_id - 1, 'cid')) if part_id else video_data.get('cid')
 
+        play_info = (
+            traverse_obj(
+                self._search_json(
+                    r'window\.__playinfo__\s*=', webpage, 'play info', video_id, default=None),
```

Comment:
line will be a little longer but it's more readable imo
```suggestion
               self._search_json(r'window\.__playinfo__\s*=', webpage, 'play info', video_id, default=None),
```

Reply:
```suggestion
                self._search_json(r'window\.__playinfo__\s*=', webpage, 'play info', video_id, default=None),
```

Reply:
sorry whitespace typo, thanks for noticing in advance